tasks/run-matlab-tests/v0/main.ts

```diff
@@ -13,7 +13,10 @@ async function run() {
             CoberturaCodeCoverage: taskLib.getInput("codeCoverageCobertura"),
             SourceFolder: taskLib.getInput("sourceFolder"),
             SelectByFolder: taskLib.getInput("selectByFolder"),
-            SelectByTag: taskLib.getInput("selectByTag")};
+            SelectByTag: taskLib.getInput("selectByTag"),
+            CoberturaModelCoverage: taskLib.getInput("coberturaModelCoverage"),
+            SimulinkTestResults: taskLib.getInput("simulinkTestResults"),
```

Comment:
Should this be `testResultsSimulinkTest` to match the style of the other test result parameter names?

Reply:
Thought of it but the test at the beginning and end sounded redundant. Updated it as `testResultsSimulinkTest`